api/client.yaml

```diff
@@ -86,6 +86,92 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/moov-io/api/master/openapi-common.yaml#/components/schemas/Error'
+  /configuration/logo:
+    get:
+      tags: [Configuration]
+      summary: Retreive an organization's logo
+      description: Retrieve a previously-uploaded logo image from an organization configuration
+      operationId: getOrganizationLogo
+      parameters:
+        - name: X-Organization
+          in: header
+          description: Value used to separate and identify models
+          example: org342
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Logo image file in original upload format
+          content:
+            image/png:
+              schema:
+                type: string
+                format: binary
+            image/jpg:
+              schema:
+                type: string
+                format: binary
+            image/svg+xml:
+              schema:
+                type: string
+                format: binary
+            image/gif:
+              schema:
+                type: string
+                format: binary
+        '400':
+          description: Failed to retrieve image file, see error(s)
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/moov-io/api/master/openapi-common.yaml#/components/schemas/Error'
+    put:
+      tags: [Configuration]
+      summary: Upload an organization's logo
+      description: Update the organization's configuration to include a logo image file.
+      operationId: uploadOrganizationLogo
```

Comment:
Since this endpoint performs an `upsert`, I recommend updating the summary/descriptions here.

Reply:
I can change it if you'd like, but the only column that actually changes is logo_file because the config it inserts is the same one that it just retrieved from the DB + LogoFile.

Reply:
Something like this could work: 
```
      summary: Upload an organization's logo, or updates it if it already exists 
      description: Updates the logo image file in the organization's configuration.
      operationId: uploadOrganizationLogo
```

Reply:
Oh I misunderstood you. That makes sense.